modules/ROOT/pages/Continuous_Delivery_Configuring_your_Continuous_Delivery_Platform.adoc

```diff
@@ -21,7 +21,7 @@ CAUTION: If your Git repository is hosted on GitHub.com, beginning August 13, 20
 
 == Maven repository credentials
 
-This job allows you to configure you access to a private Maven repository.
+This job allows you to configure your access to a private Maven repository.
 
 . Click on the image:images/JenkinsPlayButton.png[CDPlayButton] of the "Configure my maven private repo" job.
```

Comment:
Should the jenkins job not use an abbreviation for repository ?

Reply:
Yes, I changed the name of the job on Jenkins, thank you